ui/src/app/shared/pipes/safeHtml.pipe.ts

```diff
@@ -1,10 +1,23 @@
 import {Pipe, PipeTransform} from '@angular/core';
 import {DomSanitizer} from '@angular/platform-browser';
+declare var sanitizeHtml: any;
 
 @Pipe({ name: 'safeHtml'})
 export class SafeHtmlPipe implements PipeTransform  {
   constructor(private sanitized: DomSanitizer) {}
-  transform(value) {
-    return this.sanitized.bypassSecurityTrustHtml(value);
+  transform(value: string, sanitize: boolean) {
```

Comment:
Create unit test for `<script>alert("test")</script>` and `<div onclick="alert('test')"></div>`